lib/blueprinter/view_collection.rb

```diff
@@ -48,5 +53,13 @@ def sortable_fields(view_name)
 
       fields
     end
+
+    def merge_fields(source_fields, included_fields)
+      unless sort_by_definition
```

Comment:
Can we check for the positive case rather than using `unless`?

Reply:
Neither?

```
if sort_by_definition
  included_fields.merge(source_fields)
else
  source_fields.merge(included_fields)
end
```

Reply:
Sorry, I was browsing this on my mobile and carried the discussion from our previous conversation to this in my reply. 🙈 Looks straightforward, I'll change that. Do you have any other inputs/questions?